server.js

```diff
@@ -211,6 +212,12 @@ app
     server.listen(process.env.PORT || 5000, (err) => {
       if (err) throw err;
       console.log(`> Ready on /:5000`);
+      if(dev){
+        (async () => {
+          console.log("opening into browser: http://localhost:5000/");
+          await open("http://localhost:5000/");
```

Comment:
what happens if the port is not 5000? We should have a check for process.env.PORT as well.